app/qml/components/MMToolbar.qml

```diff
@@ -65,10 +65,19 @@ MMBaseToolbar {
 
   function setupBottomBar() {
     var m = root.model
-    var c = m.count
     var w = buttonView.width
     var button
 
+    for ( var j = m.count - 1 ; j >= 0 ; j-- )
```

Comment:
While you are here, would it be please possible to rename these one-letter variables to something meaningful to make the code more readable? Something like `m -> buttonModel`, `c -> buttonsCount` and `w -> buttonWidth`. Also please change `var` to `let`